public/locales/fr/translation.json

```diff
@@ -112,6 +112,27 @@
           "PlantName": "Usine"
         }
       },
+      "accordion": {
+        "nodeDetails": "Détails du nœud",
+        "findNode": {
+          "headline": "Trouver un nœud",
+          "searchByID": "Recherche par ID"
+        },
+        "exploreGraph": {
+          "headline": "Explorer un sous-graphe",
+          "startingNodes": "Sélectionner le(s) nœud(s) de départ",
+          "startingNodesError": "Sélectionnez au moins un nœud",
+          "limitDepth": "Limiter la profondeur de recherche",
+          "limitDepthError": "Entrez un nombre entier positif",
+          "flowDirection": "Choisir la direction du flux",
+          "flowDirectionError": "Sélectionnez au moins un",
+          "inEdges": "Flux entrants",
+          "outEdges": "Flux sortants",
+          "excludeEdges": "Exclure des types de relation",
+          "compoundNeighbors": "Considérer les entités des composés comme des voisins",
```

Comment:
I think it's better "Considérer les entités des composés comme voisins" without article but maybe we should ask someone else to decide :laughing:

Reply:
Yesterday, Emma and I rephrased this description to `Inlcure les autres entités de la compound`